test/rules/S3.test.ts

```diff
@@ -100,7 +96,11 @@ describe('Amazon Simple Storage Service (S3)', () => {
       });
       validateStack(stack, ruleId, TestType.NON_COMPLIANCE);
     });
-    test('Compliance', () => {
+    test('Compliance 1', () => {
+      new Bucket(stack, 'rBucket');
+      validateStack(stack, ruleId, TestType.COMPLIANCE);
+    });
+    test('Compliance 2', () => {
```

Comment:
Please collapse this into a singular test 😄

Reply:
Merged compliant tests.